[18.0][IMP] sale_stock_partner_warehouse: add option to take the warehouse from the customer or the shipping address (#4008)
OCA/sale-workflow: sale_stock_partner_warehouse/readme/CONFIGURE.md

> To set a company-wide default to take the warehouse from the shipping address:

Reviewer: ```suggestion
To set a company-wide default that prioritizes the shipping address’s warehouse (falling back to the customer’s)::
```

> Set a company-wide default to take the warehouse from the customer or the shipping address

Reviewer: ```suggestion
To set a company-wide default to take the warehouse from the customer or the shipping address:
```

- Go to Sales → Configuration → Settings

> - Enable "Sale Warehouse By Partner Shipping"

Reviewer: ```suggestion
- Enable "Prioritize Shipping Partner in Sale Warehouse Proposal"
```
Please adjust the field label accordingly.
